fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/transformation/materialization/ColumnRangePredicate.java

```diff
@@ -37,9 +37,21 @@ public class ColumnRangePredicate extends RangePredicate {
     // the relation between each Range in RangeSet is 'or'
     private TreeRangeSet<ConstantOperator> columnRanges;
```

Comment:
still need this?

Reply:
yes, use the original column range to get the final ScalarOperator.